CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 PROJECT(tntcxx)
+include(CTest)
```

Comment:
Although cmake is not case sensitive it's not nice when one file is written in different case. Please make it all upper-case.
There's no such thing in our code style but imho it should be.

Reply:
Sure, this slipped out unintentionally, sorry.